test-cases/scylla-operator/kubernetes-scylla-upgrade.yaml

```diff
@@ -22,9 +22,10 @@ user_prefix: 'kubernetes-scylla-upgrade'
 #authenticator_user: 'cassandra'
 #authenticator_password: 'cassandra'
 
-# Does not work due to the lack of support, could be anabled after - https://github.com/scylladb/scylla-cluster-tests/issues/2747
-#append_scylla_args: '--blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc --abort-on-internal-error 1'
-
+# TODO: add '--abort-on-seastar-bad-alloc' arg to the 'append_scylla_args' option when
+#       following operator bug gets fixed: https://github.com/scylladb/scylla-operator/issues/991
+#       '--blocked-reactor-notify-ms 100' cannot be set, because it gets set by operator itself
+append_scylla_args: '--abort-on-lsa-bad-alloc 1 --abort-on-internal-error 1 --abort-on-ebadf 1 --enable-sstable-key-validation 1'
```

Comment:
It is already defined in each of the K8S backends default config files.
So, if change this file then only by removing the red lines here.

Reply:
ok